source/Dig/base.gfx.gui.button.bmx

```diff
@@ -155,39 +157,50 @@ endrem
 	Method GetSprite:TSprite()
 		Local newSprite:TSprite
 
-		If Not IsEnabled() 
+		If Not IsEnabled()
 			If _spriteNameInUse <> _spriteNameDisabled
-				newSprite = GetSpriteFromRegistry(_spriteNameDisabled, _sprite)
+				newSprite = GetSpriteFromRegistry(_spriteNameDisabled)
+				'if not defined, try to fall back to "normal" one
+				if newSprite.name = "defaultsprite" Then newSprite = GetSpriteFromRegistry(_spriteName)
```

Comment:
Die Fallbacks zu normal sind in allen Fällen identisch, oder?
Könnte das dann generisch im Block Zeile 201 erledigt werden?

Reply:
Jein ... also die sind "zufaellig" die gleichen. Wenn da Einsprueche sind ("nee also disabled muss _immer_ anders aussehen als der normale Button") dann wuerde man das dort rausnehmen koennen.

Reply:
OK. Dann so mergen.